File: pos-soap-node/consumerPessoa.js
var soap = require('soap');
var url = 'http://localhost:8080/pos-soap-server/ServicePessoaService?wsdl';
var pessoa = {'arg0':{
  'id': '0',
  'nome':'Nhonho'
}};
soap.WSDL.prototype.ignoredNamespaces = ['http://ws.pos.ifpb.edu.br/'];
soap.createClient(url, function(err, client) {
    client.salvar(pessoa,function(err,result,res){
         console.log('Retorno:',res);
         client.todasAsPessoas(function(err,result,res){
              console.log(result.return);
         })
    })
    // client.todasAsPessoas(function(err,result,res){
    //      console.log(result.return);
    //     //  console.log('Retorno:',result);
    // })
});

File: pos-soap-python/consumerPessoa.py
from suds.client import Client
from suds.cache import NoCache
url="http://localhost:8080/pos-soap-server/ServicePessoaService?wsdl"
client = Client(url, cache=NoCache())
pessoa = {
    'id': '0',
    'nome':'Florinda'
}
#invocando os metodos
client.service.salvar(pessoa);
print client.service.todasAsPessoas();
#print client
